js/exporter/exceljs/export_pivot_grid.js

```diff
@@ -3,22 +3,27 @@ import { Export } from './export';
 import { noop } from '../../core/utils/common';
 
 const privateOptions = {
-    _getWorksheetFrozenState: function(dataProvider, cellRange) {
+    _getWorksheetFrozenState(dataProvider, cellRange) {
         return { state: 'frozen', xSplit: cellRange.from.column + dataProvider.getFrozenArea().x - 1, ySplit: cellRange.from.row + dataProvider.getFrozenArea().y - 1 };
     },
 
-    _getCustomizeCellOptions: function(excelCell, pivotCell) {
+    _getCustomizeCellOptions(excelCell, pivotCell) {
         return {
             excelCell: excelCell,
             pivotCell: pivotCell
         };
     },
 
-    _needMergeRange: function() {
+    _needMergeRange() {
         return true;
     },
 
-    _renderLoadPanel: function(component) {
+    _isRangeMerged(dataProvider, rowIndex, cellIndex, rowspan, colspan, mergeRowFieldValues, mergeColumnFieldValues) {
```

Comment:
Result of this function doesn't look as "whether this range is merged?"
It looks like "is it a root cell of a merged region?"

Reply:
We deside that cell is a master cell the range with use this condition (colspan || rowspan) before the method isRangeMerged() called

Reply:
Method name/result should not depend on the outer code.
Based on this expression our code decides whether 'this.mergeRanges.push' should be called or not.
Can we rename it to _canAddToMergedRanges?
Or, it can mean _canHandleRangeAsMerged.
Or, it can mean _handleMergedCellsAsUnmerged.

Reply:
Renamed to _allowToMergeRange